src/util-exception-policy.c

```diff
@@ -32,6 +32,10 @@ void ExceptionPolicyApply(Packet *p, enum ExceptionPolicy policy, enum PacketDro
         switch (policy) {
             case EXCEPTION_POLICY_IGNORE:
                 break;
+            case EXCEPTION_POLICY_REJECT:
+                SCLogDebug("EXCEPTION_POLICY_REJECT");
+                PacketDrop(p, ACTION_REJECT, drop_reason);
```

Comment:
will calling `PacketDrop` twice in this path have any side effects?

Reply:
I've considered that... Since now `PacketDrop` uses `PACKET_UPDATE_ACTION` and this one will not override the existing action or drop reason, my understanding is that it should be ok...